openwisp_radius/tests/test_social.py

```diff
@@ -66,8 +66,8 @@ def test_redirect_cp_301(self):
             u.username, token.key, rad_token.key
         )
         self.assertIn(querystring, r.url)
-        u = User.objects.filter(username='socialuser').first()
-        self.assertIn((self.default_org.pk,), u.organizations_pk)
+        user = User.objects.filter(username='socialuser').first()
+        self.assertTrue(user.organizations_dict.get(str(self.default_org.pk), False))
```

Comment:
Unrelated, change because `organizations_pk` is deprecated.

Reply:
why not:
```python
self.assertTrue(user.is_member(self.default_org))
self.assertFalse(user.is_manager(self.default_org))
```

Reply:
why not:
```python
self.assertTrue(user.is_member(self.default_org))
self.assertFalse(user.is_manager(self.default_org))
```